clients/rust/src/lib.rs

```diff
@@ -26,8 +28,8 @@ use protobuf::Message;
 use byteorder::{WriteBytesExt, LittleEndian};
 use protobuf::repeated::RepeatedField;
 
-// wildcard is recommended for error-chain
-pub use error_chain_generated_errors::*;
+use failure::ResultExt;
+use errors::{SysadminErrorKind as ErrorKind, SysadminResult};
```

Comment:
To me, this suffers the same issues as shadowing the built in Result type. I think it's clearer to directly refer to the type instead of using this alias.